src/Raven.Server/Rachis/RachisLogHistory.cs

```diff
@@ -489,5 +489,106 @@ public unsafe bool HasHistoryLog(ClusterOperationContext context, BlittableJsonR
                 return true;
             }
         }
+
+        public static long? TryGetLastRaftIndexForDeleteDatabaseCommandWithoutErrors(ServerStore serverStore, string databaseName, out bool hasExceptions)
+        {
+            var commandType = nameof(DeleteDatabaseCommand);
+            long? index = null;
+            hasExceptions = false;
+
+            using (serverStore.Engine.ContextPool.AllocateOperationContext(out ClusterOperationContext context))
+            using (context.OpenReadTransaction())
+            {
+                var table = context.Transaction.InnerTransaction.OpenTable(LogHistoryTable, LogHistorySlice);
+                var guidPrefix = $"{databaseName}/";
+                using (Slice.From(context.Allocator, guidPrefix, out var guidPrefixSlice))
+                {
+                    foreach (var (guidSlice, entryHolder) in table.SeekByPrimaryKeyPrefix(guidPrefixSlice, Slices.Empty, 0))
```

Comment:
you will seek _all_ commands of the database, can we seek only for the deletion commands?
to have a prefix like `<database>/delete/`

Reply:
I have to search it like this:
`table.SeekBackwardFrom(LogHistoryTable.Indexes[LogHistoryIndexSlice], Slices.AfterAllKeys)`,
else they wont be ordered by index